apps/clock_info/lib.js

```diff
@@ -333,6 +336,7 @@ exports.addInteractive = function(menu, options) {
   menuShowItem(menu[options.menuA].items[options.menuB]);
   // return an object with info that can be used to remove the info
   options.remove = function() {
+    save();
```

Comment:
we'd need to do `E.removeListener("kill",save)` here too I reckon or we will have a memory leak when doing fast load

Reply:
Good catch, thanks!